streetsense/docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
   backend:
     build:
       context: ./streetsense_backend
-    command: python manage.py runserver 0.0.0.0:8000
+    command: sh -c "python manage.py migrate && python manage.py runserver 0.0.0.0:8000"
```

Comment:
shouldn't migrate happen after the runserver command? I got errors when it was this way.

Reply:
Have you ran makemigrations once with the docker containers running? I made a change to the CrimeReport table and I think django needs to generate a report every time that happens. I forgot to add the report file, so it is the 2nd commit to this push, if you pull again you should have it, but you might still need to make migrations. 

This made the report for me:

    docker-compose exec backend python manage.py makemigrations 

I tried switching the order of migrate and manage and it won't work for me.

Reply:
yea I had the newest commit pulled. But it works now! idk why

Reply:
I still get errors in the backend when running migrate before run-server though 
<img width="1082" alt="Screenshot 2024-10-26 at 14 55 28" src="https://github.com/user-attachments/assets/a574d632-3d9d-4477-b7d9-64c2f3c27d00">

Reply:
I think running both commands at once might happen before the database is actually up and running

Reply:
as the manage.py migrate command is only for us devs when we make a change to the database, can't we just run it manually instead?

Reply:
Yea that might be for the best.